src/app/core/models/templates/sections/common-parsers.model.ts

```diff
@@ -1,14 +1,17 @@
 const { char, choice, coroutine, everyCharUntil, str, whitespace } = require('arcsecond');
 
 const TEAM_RESPONSE_HEADER = "# Team's Response";
+const DEFAULT_TEAM_RESPONSE = 'No details provided by team.';
 
 export function buildTeamResponseSectionParser(nextHeader: string) {
   return coroutine(function* () {
     yield str(TEAM_RESPONSE_HEADER);
     yield whitespace;
-    const teamResponse = yield everyCharUntil(str(nextHeader));
+    let teamResponse = yield everyCharUntil(str(nextHeader));
 
-    return teamResponse.trim();
+    teamResponse = teamResponse.trim();
+    teamResponse = teamResponse === '' ? DEFAULT_TEAM_RESPONSE : teamResponse;
+    return teamResponse;
```

Comment:
Looks like this would return a boolean, but
```typescript
const teamResponse = yield everyCharUntil(str(nextHeader));
return teamResponse.trim() ? teamResponse.trim() : DEFAULT_TEAM_RESPONSE;
```
works